### PR TITLE
wizard fixup

### LINKED ISSUE #53: Wizard doesn't run following instructions
Started on a fresh install and followed the instructions the wizard was built but never executed.

### PR DESCRIPTION
**Pull request by jrabbit**
Retool dockerfile to stick to 3.7 & add maintainer, change wizard to not default always to "app", run when .env exists (like it does in the git tree)
This fixes #53

### CODE REVIEW

**Comment by tsileo** [comment]
Hey, thanks for the PR, I have a few questions, I will comment on the diff.

Thank you.

File: setup_wizard/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
-FROM python:3
+FROM python:3.7
 WORKDIR /app
 ADD . /app
 RUN pip install -r requirements.txt
+LABEL maintainer="t@a4.io"
+LABEL pub.microblog.oneshot=true
```
Comment (tsileo) [inline setup_wizard/Dockerfile]: Is this kind of label a "standar", or it's just informative? I have not been able found any information.

File: Makefile [hunk 2]
```diff
@@ -2,13 +2,13 @@ PYTHON=python
...
 
-# Build the config (will error if an existing config is found) via a Docker container
+# Build the config (will error if an existing config/me.yml is found) via a Docker container
 .PHONY: config
 config:
 	# Build the container for the setup wizard on-the-fly
 	cd setup_wizard && docker build . -t $(SETUP_WIZARD_IMAGE)
 	# Run and remove instantly
-	-docker run --rm -it --volume $(PWD):/app/out $(SETUP_WIZARD_IMAGE)
+	-docker run -e MICROPUB_WIZARD_PROJECT_NAME --rm -it --volume $(PWD):/app/out $(SETUP_WIZARD_IMAGE)
```
Comment (tsileo) [inline Makefile]: the env var name should be `MICROBLOGPUB_WIZARD_PROJECT_NAME`.

[Micropub](https://www.w3.org/TR/micropub/) is another standard.

File: setup_wizard/wizard.py [hunk 3]
```diff
@@ -58,12 +57,14 @@ def main() -> None:
     with config_file.open("w") as f:
         f.write(out)
 
+    proj_name = os.getenv("MICROPUB_WIZARD_PROJECT_NAME", "micropub")
```
Comment (tsileo) [inline setup_wizard/wizard.py]: The default project name should be `microblogpub`: the default path for the repo will be `microblog.pub` and docker-compose removes the dot in the dir name.

File: setup_wizard/Dockerfile (hunk 1 above)
Comment (jrabbit) [reply setup_wizard/Dockerfile]: Yeah maintainer is best practices, the other is namespaced and just descriptive.

File: setup_wizard/wizard.py (hunk 3 above)
Comment (jrabbit) [reply setup_wizard/wizard.py]: The current default is always "app" even though it checks every time there's no other possible setting unless you re-write the wizard dockerfile by hand ahead of time

**Comment by jrabbit** [comment]
@tsileo anything else needed (new changes above)

**Comment by tsileo** [comment]
Thanks!

**Pull request merged**